Makefile

```
@@ -192,3 +193,17 @@ scheduler-plugins-chart: scheduler-plugins-crd
 	cp -f $(PROJECT_DIR)/dep-crds/scheduler-plugins/crd.yaml $(PROJECT_DIR)/dep-manifests/scheduler-plugins/crds/scheduling.x-k8s.io_podgroups.yaml
 	cp -f /tmp/pkg/mod/sigs.k8s.io/scheduler-plugins@$(SCHEDULER_PLUGINS_VERSION)/manifests/noderesourcetopology/crd.yaml $(PROJECT_DIR)/dep-manifests/scheduler-plugins/crds/topology.node.k8s.io_noderesourcetopologies.yaml
 	chmod -R 760 $(PROJECT_DIR)/dep-manifests/scheduler-plugins
+
+.PHONY: volcano-scheduler
+volcano-scheduler:
+	-@GOPATH=/tmp go install volcano.sh/volcano/cmd/scheduler@$(VOLCANO_SCHEDULER_VERSION)
+
+.PHONY: volcano-scheduler-crd
+volcano-scheduler-crd: volcano-scheduler
+	mkdir -p $(PROJECT_DIR)/dep-crds/volcano-scheduler/
+	cp -f /tmp/pkg/mod/volcano.sh/volcano@$(VOLCANO_SCHEDULER_VERSION)/config/crd/bases/* $(PROJECT_DIR)/dep-crds/volcano-scheduler
```

```suggestion
VOLCANO_ROOT = $(shell go list -m -f "{{.Dir}}" volcano.sh/volcano)
.PHONY: volcano-scheduler-crd
volcano-scheduler-crd:
	mkdir -p $(shell pwd)/dep-crds/volcano-scheduler/
	cp -f $(VOLCANO_ROOT)/config/crd/bases/*  $(shell pwd)/dep-crds/volcano-scheduler/
```

Did this approach not work?

---

we use `volcano.sh/apis` in go.mod for its clientsets api, not `volcano.sh/volcano`. But e2e test need to install yamls provided in `volcano.sh/volcano`

---

Ah, I see. That makes sense.
Thanks!